### DISCUSSION TITLE
Is the language still in active development?

### DISCUSSION THREAD

**Discussion by treejadey** [mint-lang/mint#553] [Answered] [answered]
As the title says, I am wondering if the language is still being worked on. I really like the language and it would be a big shame if it is abandoned.

**Comment by iamwacko** [comment]
"It's not in active development, but it's alive 🙂" - @gdotdesign on discord

**Comment by gdotdesign** [comment]
Mint is something that I still want to get to a point where it's production ready (let's say 1.0), however I'm using it in production in [DevBox](https://www.dev-box.app/) which is ~50K LOC and works well but have some things I would like to work on and make easier.

I haven't worked on it in a while and just started again by refactoring and separating the type system https://github.com/mint-lang/hm-type-system which is long overdue and a blocker for a number of features (https://github.com/mint-lang/mint/pull/283 for example).

I can't really promise anything timing wise at this point, but for the next few months my plan is to:
- Finish and merge the syntax change PRs (#532, #503, #528)
- Finish and integrate https://github.com/mint-lang/hm-type-system

After that, I have some plans to refactor the compiler to be able to do code-splitting and dynamic loading, allow some form of polymorphism (interface, trait or something along those lines) and more work on the language server.

Please note that I'm only working on Mint in the spare time I have left after work, DevBox and my family :slightly_smiling_face: If you have any questions just ask :+1:

**Comment by gdotdesign** [comment, marked as answer]
I've just released 0.20.0 so it's alive!
